op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -113,20 +233,44 @@ func Test_Operator_Fee_Constistency(gt *testing.T) {
 		env.RunFaultProofProgram(t, l2SafeHead.Number, testCfg.CheckResult, testCfg.InputParams...)
 	}
 
-	matrix := helpers.NewMatrix[any]()
+	matrix := helpers.NewMatrix[testCase]()
 	defer matrix.Run(gt)
 
 	matrix.AddTestCase(
-		"HonestClaim-OperatorFeeConstistency",
-		nil,
+		"HonestClaim-OperatorFeeConstistency-NormalTx",
+		NormalTx,
+		helpers.NewForkMatrix(helpers.Isthmus),
+		runIsthmusDerivationTest,
+		helpers.ExpectNoError(),
+	)
+
+	matrix.AddTestCase(
```

Comment:
I think you can daisy chain these together, and even use `matrix.AddDefaultCases` to tidy this up a bit. There's ` matrix.AddDefaultCasesWithName` in flight here https://github.com/ethereum-optimism/optimism/pull/14006/commits/ac655a37897032fce3c784f0780beb835bad017b (should be merged soon).

Reply:
it's merged, would indeed be great to use it here